crates/core/tedge_api/src/device_profile.rs

```diff
@@ -41,6 +41,7 @@ impl CommandPayload for DeviceProfileCmdPayload {
 pub struct DeviceProfileOperation {
     #[serde(flatten)]
     pub operation: OperationPayload,
+    #[serde(default, rename = "@skip")]
```

Comment:
I don't think we can do that now as this has been part of a public API, esp a mandatory field. So, if we change it now, it might affect existing users.

Reply:
Scratch that. It is indeed `@skip` that the agent supports: https://github.com/thin-edge/thin-edge.io/blob/e509ae6bf27526f03a45c9c3e74cfa11f40c03df/crates/core/tedge_api/src/workflow/mod.rs#L414